src/main.rs

```diff
@@ -166,12 +191,61 @@ struct ConfigPath {
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 struct DaftConfig {
     setup: DaftSetup,
-    #[serde(default)]
-    run: Vec<StrRef>,
-    #[serde(default, rename = "job", deserialize_with = "parse_jobs")]
+    #[serde(rename = "job", deserialize_with = "parse_jobs")]
```

Comment:
We will probably want to have `#[serde(default, ...)]`, since without it, deserialization will fail if a `[[job]]` is not present. Although the default configuration file that is outputted by `daft init` contains a job, a configuration file *without* any jobs should also be valid.

Reply:
This brings up another good test case. We don't have anything that tests deserialization for non-default files. Our current tests only test that the default configuration file that is outputted by `daft init` is good, but that doesn't test other valid configuration files.